diff --git a/setup/distfiles/inventory/iocage.py b/setup/distfiles/inventory/iocage.py
index bd479ae..7f0622d 100644
--- a/setup/distfiles/inventory/iocage.py
+++ b/setup/distfiles/inventory/iocage.py
@@ -9,7 +9,7 @@ from __future__ import annotations
 DOCUMENTATION = r'''
     name: iocage
     short_description: iocage inventory source
-    version_added: 10.2.0
+    version_added: 0.3.0
     author:
         - Vladimir Botka (@vbotka)
     requirements:
@@ -29,10 +29,10 @@ DOCUMENTATION = r'''
         plugin:
             description:
                 - The name of this plugin, it should always be set to
-                  V(community.general.iocage) for this plugin to recognize
+                  V(vbotka.freebsd.iocage) for this plugin to recognize
                   it as its own.
             required: true
-            choices: ['community.general.iocage']
+            choices: ['vbotka.freebsd.iocage']
             type: str
         host:
             description: The IP/hostname of the C(iocage) host.
@@ -51,14 +51,14 @@ DOCUMENTATION = r'''
                 - This requires passwordless sudo of the command C(iocage list*).
             type: bool
             default: false
-            version_added: 10.3.0
+            version_added: 0.5.0
         sudo_preserve_env:
             description:
                 - Preserve environment if O(sudo) is enabled.
                 - This requires C(SETENV) sudoers tag.
             type: bool
             default: false
-            version_added: 10.3.0
+            version_added: 0.5.0
         get_properties:
             description:
                 - Get jails' properties.
@@ -79,7 +79,7 @@ DOCUMENTATION = r'''
                 - The variable C(iocage_hooks) is not created if O(hooks_results) is empty.
             type: list
             elements: path
-            version_added: 10.4.0
+            version_added: 0.5.2
     notes:
         - You might want to test the command C(ssh user@host iocage list -l) on
           the controller before using this inventory plugin with O(user) specified
@@ -96,17 +96,17 @@ DOCUMENTATION = r'''
 EXAMPLES = r'''
 ---
 # file name must end with iocage.yaml or iocage.yml
-plugin: community.general.iocage
+plugin: vbotka.freebsd.iocage
 host: 10.1.0.73
 user: admin
 
 ---
 # user is not required if iocage is running on localhost (default)
-plugin: community.general.iocage
+plugin: vbotka.freebsd.iocage
 
 ---
 # run cryptography without legacy algorithms
-plugin: community.general.iocage
+plugin: vbotka.freebsd.iocage
 host: 10.1.0.73
 user: admin
 env:
@@ -115,7 +115,7 @@ env:
 ---
 # execute as root
 # sudoers example 'admin ALL=(ALL) NOPASSWD:SETENV: /usr/local/bin/iocage list*'
-plugin: community.general.iocage
+plugin: vbotka.freebsd.iocage
 host: 10.1.0.73
 user: admin
 sudo: true
@@ -125,7 +125,7 @@ env:
 
 ---
 # enable cache
-plugin: community.general.iocage
+plugin: vbotka.freebsd.iocage
 host: 10.1.0.73
 user: admin
 env:
@@ -134,7 +134,7 @@ cache: true
 
 ---
 # see inventory plugin ansible.builtin.constructed
-plugin: community.general.iocage
+plugin: vbotka.freebsd.iocage
 host: 10.1.0.73
 user: admin
 env:
@@ -154,7 +154,7 @@ keyed_groups:
 
 ---
 # Read the file /var/db/dhclient-hook.address.epair0b in the jails and use it as ansible_host
-plugin: community.general.iocage
+plugin: vbotka.freebsd.iocage
 host: 10.1.0.73
 user: admin
 hooks_results:
@@ -204,7 +204,7 @@ def _parse_ip4(ip4):
 class InventoryModule(BaseInventoryPlugin, Constructable, Cacheable):
     ''' Host inventory parser for ansible using iocage as source. '''
 
-    NAME = 'community.general.iocage'
+    NAME = 'vbotka.freebsd.iocage'
     IOCAGE = '/usr/local/bin/iocage'
 
     def __init__(self):
