diff --git a/setup/distfiles/inventory/iocage.py b/setup/distfiles/inventory/iocage.py
index 7575bd2..9001c33 100644
--- a/setup/distfiles/inventory/iocage.py
+++ b/setup/distfiles/inventory/iocage.py
@@ -10,7 +10,7 @@ __metaclass__ = type
 DOCUMENTATION = '''
     name: iocage
     short_description: iocage inventory source
-    version_added: 10.2.0
+    version_added: 0.5.0
     author:
         - Vladimir Botka (@vbotka)
     requirements:
@@ -30,10 +30,10 @@ DOCUMENTATION = '''
         plugin:
             description:
               - The name of this plugin, it should always be set to
-                V(community.general.iocage) for this plugin to recognize
+                V(vbotka.freebsd.iocage) for this plugin to recognize
                 it as its own.
             required: true
-            choices: ['community.general.iocage']
+            choices: ['vbotka.freebsd.iocage']
             type: str
         host:
             description: The IP/hostname of the C(iocage) host.
@@ -78,15 +78,15 @@ DOCUMENTATION = '''
 
 EXAMPLES = '''
 # file name must end with iocage.yaml or iocage.yml
-plugin: community.general.iocage
+plugin: vbotka.freebsd.iocage
 host: 10.1.0.73
 user: admin
 
 # user is not required if iocage is running on localhost (default)
-plugin: community.general.iocage
+plugin: vbotka.freebsd.iocage
 
 # run cryptography without legacy algorithms
-plugin: community.general.iocage
+plugin: vbotka.freebsd.iocage
 host: 10.1.0.73
 user: admin
 env:
@@ -95,7 +95,7 @@ env:
 # execute as root
 # this requires sudoers configuration on the iocage host
 # for example: admin ALL=(ALL) NOPASSWD:SETENV: /usr/local/bin/iocage list*
-plugin: community.general.iocage
+plugin: vbotka.freebsd.iocage
 host: 10.1.0.73
 user: admin
 sudo: true
@@ -103,7 +103,7 @@ env:
   CRYPTOGRAPHY_OPENSSL_NO_LEGACY: 1
 
 # enable cache
-plugin: community.general.iocage
+plugin: vbotka.freebsd.iocage
 host: 10.1.0.73
 user: admin
 env:
@@ -111,7 +111,7 @@ env:
 cache: true
 
 # see inventory plugin ansible.builtin.constructed
-plugin: community.general.iocage
+plugin: vbotka.freebsd.iocage
 host: 10.1.0.73
 user: admin
 env:
@@ -169,7 +169,7 @@ def _parse_ip4(ip4_addr):
 class InventoryModule(BaseInventoryPlugin, Constructable, Cacheable):
     ''' Host inventory parser for ansible using iocage as source. '''
 
-    NAME = 'community.general.iocage'
+    NAME = 'vbotka.freebsd.iocage'
     IOCAGE = '/usr/local/bin/iocage'
 
     def __init__(self):
