--- ucl.py.orig	2026-08-30 09:18:25.355735228 +0200
+++ ucl.py	2026-08-30 08:18:58.991992334 +0200
@@ -130,23 +130,23 @@
 
 EXAMPLES = r"""
 - name: Get FreeBSD repository URL
-  ucl:
+  vbotka.freebsd.ucl:
     path: /etc/pkg/FreeBSD.conf
     upath: freebsd.url
 
 - name: Get package configuration in YAML format
-  ucl:
+  vbotka.freebsd.ucl:
     path: /etc/pkg/FreeBSD.conf
     lang: yaml
 
 - name: Set latest package repository URL
-  ucl:
+  vbotka.freebsd.ucl:
     path: /etc/pkg/FreeBSD.conf
     upath: freebsd.url
     value: "pkg+http://pkg.FreeBSD.org/${ABI}/latest"
 
 - name: Merge new value to key
-  ucl:
+  vbotka.freebsd.ucl:
     path: /usr/local/etc/pkg/repos/custom.conf
     upath: Custom
     value:
@@ -156,7 +156,7 @@
     create: true
 
 - name: Remove key from config
-  ucl:
+  vbotka.freebsd.ucl:
     path: /foo/bar.conf
     upath: rootkey.subkey
     state: absent
